worker/crates/action-universal/src/attribute/renamer.rs

```diff
@@ -20,30 +23,75 @@ pub(super) enum Rename {
 #[derive(Serialize, Deserialize, Debug, Clone)]
 #[serde(rename_all = "camelCase")]
 pub(super) enum RenameAction {
-    AddStringPreffix(String),
+    AddStringPrefix(String),
     AddStringSuffix(String),
     RemovePrefixString(String),
     RemoveSuffixString(String),
+    StringReplace(ReplaceString),
+    RegularExpressionReplace(ReplaceString),
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+#[serde(rename_all = "camelCase")]
+pub(super) struct ReplaceString {
+    from: String,
+    to: String,
 }
 
 #[async_trait::async_trait]
 #[typetag::serde(name = "BulkAttributeRenamer")]
-impl Action for BulkAttributeRenamer {
+impl AsyncAction for BulkAttributeRenamer {
     async fn run(&self, _ctx: ActionContext, inputs: Option<ActionDataframe>) -> ActionResult {
         let output = inputs
+            .ok_or(Error::input("no input"))?
+            .into_iter()
+            .map(|(k, v)| {
+                (
+                    if k == *DEFAULT_PORT {
+                        k
+                    } else {
+                        Port::new(rename_key(k.into_inner(), &self.action))
```

Comment:
Oh, this is different.

Reply:
@kirisaki 
By attribute, we mean a column of table data.
In other words, this function renames the hashmap key of Array<HashMap>.